content/blog/self-hosted-search-and-deploy/index.md

```diff
@@ -0,0 +1,60 @@
+
+## Self-Hosted Pulumi Cloud Overview
+
+Self-Hosting is available in the Pulumi Business Critical edition of Pulumi Cloud, you can try it out with [our 30 day trial](/product/self-hosted). This allows organizations to run the Pulumi Cloud within their own environments, ensuring data control, compliance, and customizability. While Pulumi manages all aspects of its SaaS offering (Pulumi Cloud), Self-Hosted Pulumi Cloud requires customers to handle data backups, service maintenance, and updates.
+
+As of today, all Pulumi Cloud features that have been made generally available are supported in the self-hosted version.
+
+Here’s a deep dive into what the new features offer and how they can benefit your organization:
+
+## Resource Search: Find and Analyze Your Cloud Resources with Ease
+
+Pulumi Resource Search offers multi-cloud search and analytics across every environment in your organization. You can issue queries that find all of your AWS VPCs, or all of your VPCs in AWS and Azure, or all resources with the “production” tag across all cloud environments. You also have access to statistics about cloud usage, including a breakdown by cloud provider, resource type, and department. Resource search enables you to find the needle within your cloud haystack and visualize cloud consumption trends.
+
+![resource search UI](search.png)
+
+### Key Features
+
+- **Multi-Cloud Search:** Query resources across various cloud environments.
+- **Advanced Filtering:** See top stacks, projects, providers, teams, and types by resource count.
+- **AI Assist:** Use natural language queries to generate search syntax.
+- **Search API:** Integrate search functionality into your internal systems using the Search API.
+- **Search from the CLI:** Issue resource search queries from within the terminal without needing to change contexts.
+- **Data Export:** Export resource data for further analysis in your data warehouse.
+
+## Pulumi Deployments: Automate and Manage Your Infrastructure with Ease
+
+Pulumi Deployments is engineered to streamline infrastructure management tasks with a focus on scalability, automation, and control. This platform allows cloud engineers and platform teams to execute infrastructure actions such as previews, updates, destroys, and refreshes directly within Pulumi’s managed service. By bundling deployment settings like source code, cloud credentials, and environment variables into a single unit, Pulumi Deployments eliminates the complexities of stack management. Users can trigger deployments via Git events, drift being detected, through the Pulumi Cloud UI or leverage REST APIs for custom workflows, providing the flexibility to efficiently transition from code to cloud.
```

Comment:
drift being detected feels awkward in this context, and is not technically true as a trigger.  Maybe add another line like "In addition, you can schedule deployments to detect drift or enforce a time to live (TTL)."